contrib/zellij/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "zellij"
+pkgver = "0.40.1"
+pkgrel = 0
+build_style = "cargo"
+make_check_args = ["--release"]
```

Comment:
i think we don't want this?

Reply:
it fails without it because of the way debug/release interacts with whether plugins are wasm or not

Reply:
feel free to remove and figure out the other way thought :P  

might also just make sense to skip tests for this, dunno if they test something useful

Reply:
ah ok

Reply:
would be nice to add a comment for that